src/logion/model/locrequest.model.ts

```diff
@@ -218,11 +227,20 @@ export class EmbeddableLifecycle {
         }
     }
 
-    static from(alreadyReviewed: boolean) {
+    static fromSubmissionType(submissionType: SubmissionType) {
         const lifecycle = new EmbeddableLifecycle();
-        lifecycle.status = alreadyReviewed ? "REVIEW_ACCEPTED" : "DRAFT";
-        lifecycle.acknowledgedByOwner = alreadyReviewed;
-        lifecycle.acknowledgedByOwnerOn = alreadyReviewed ? moment().toDate() : undefined;
+        lifecycle.status =
+            submissionType === "DIRECT_BY_REQUESTER" ? "PUBLISHED" :
+                submissionType === "MANUAL_BY_OWNER" ? "REVIEW_ACCEPTED" : "DRAFT";
+        lifecycle.acknowledgedByOwner = submissionType === "MANUAL_BY_OWNER";
+        lifecycle.acknowledgedByOwnerOn = submissionType === "MANUAL_BY_OWNER" ? moment().toDate() : undefined;
+        lifecycle.acknowledgedByVerifiedIssuer = false;
+        return lifecycle;
+    }
+
+    static default() {
```

Comment:
Why not set the status in this function?

Reply:
Done, but it does not make the code shorter.